File: Angular/satva/src/app/onlinepayment.ts
export class Onlinepayment {
    private paymentMode: string = '';
    public get paymentmode(): string {
        return this.paymentMode;
    }
    public set paymentmode(value: string) {
        this.paymentMode = value;
    }
	private cardType: string = '';
    public get cardtype(): string {
        return this.cardType;
    }
    public set cardtype(value: string) {
        this.cardType = value;
    }
	private paymentDate:Date|null = null;
    public get paymentdate(): Date|null {
        if(this.paymentDate!=null)
        return this.paymentDate;
        else
        return null
    }
    public set paymentdate(value: Date|null) {
        this.paymentDate = value;
    }
	private amount: number = 0;
    public get amountValue(): number {
        return this.amount;
    }
    public set amountValue(value: number) {
        this.amount = value;
    }
}

File: SpringBoot/demo/src/main/java/com/example/rps/beans/buy/OnlinePayment.java
package com.example.rps.beans.buy;

import java.time.LocalDateTime;

import com.example.rps.beans.LocalDateTimeDeserializer;
import com.example.rps.beans.LocalDateTimeSerializer;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;

public class OnlinePayment {
	
	private String paymentMode;
	private String cardType;
	
	 @JsonSerialize(using = LocalDateTimeSerializer.class)
	 @JsonDeserialize(using = LocalDateTimeDeserializer.class)
	private LocalDateTime paymentDate;
	private float amount;
	
	public String getPaymentMode() {
		return paymentMode;
	}
	public void setPaymentMode(String paymentMode) {
		this.paymentMode = paymentMode;
	}
	public String getCardType() {
		return cardType;
	}
	public void setCardType(String cardType) {
		this.cardType = cardType;
	}
	public LocalDateTime getPaymentDate() {
		return paymentDate;
	}
	public void setPaymentDate(LocalDateTime paymentDate) {
		this.paymentDate = paymentDate;
	}
	public float getAmount() {
		return amount;
	}
	public void setAmount(float amount) {
		this.amount = amount;
	}
	

}
